examples/test/GeoPushpinChart_test.js

```diff
@@ -0,0 +1,61 @@
+// (C) 2020 GoodData Corporation
+import { Selector } from "testcafe";
+import { config } from "./utils/config";
+import { loginUsingLoginForm } from "./utils/helpers";
+import { DEFAULT_COLOR_PALETTE } from "../src/utils/colors";
+
+fixture("Geo Pushpin Chart Demo") // eslint-disable-line no-undef
+    .page(config.url)
+    .beforeEach(loginUsingLoginForm(`${config.url}/geo-pushpin-chart`));
+
+test("should render all charts", async t => {
+    const geoPushpinChartWithClustering = Selector(".s-geo-pushpin-chart-clustering");
+    const geoPushpinChartWithColorLegend = Selector(".s-geo-pushpin-chart-color");
+    const geoPushpinChartWithCategoryLegend = Selector(".s-geo-pushpin-chart-category");
+    const geoPushpinChartWithConfiguration = Selector(".s-geo-pushpin-chart-configuration");
```

Comment:
These constants should be defined global scope so it can be re-used

Reply:
fixed